goverge/coverage.py

```diff
@@ -201,11 +201,12 @@ def get_package_deps(project_package, test_path, tag):
         stdout=subprocess.PIPE, cwd=test_path).communicate()
 
     output = test_output.split() + output.split()
-
-    package_deps = [
-        package.replace("]", "").replace("[", "").replace("'", "")
-        for package in list(set(output))
-        if project_package.split("\n")[0] in package]
+    project_path = project_package.split("\n")[0]
+    package_deps = []
+    for package in list(set(output)):
+        if project_path in package and project_path + "/vendor" not in package:
```

Comment:
This reads backwards to me, it's checking if the project path is in package, but then verifying that the vendor path is _not_ in the package?

I'm not really sure what this is doing here still - maybe I'm missing something? can the docstring for this method be more clear? or a comment here?

I trust it works as expected since it seems to pass the tests, but it's not clear to me why you are excluding the vendor directory (now that I can understand the logic :) ).